packages/openactive-integration-tests/test/templates/common.js

```diff
@@ -18,21 +17,22 @@ const shortid = require('shortid');
  */
 
 /**
- * @param {boolean} [includeIdentifier]
+ * @param {string | null | undefined} [identifier]
```

Comment:
Do we need to add more detail to this comment to explain the difference between passing `null` and `undefined` here?